script/integration.sh

```diff
@@ -0,0 +1,3 @@
+npm start &
+wait-on http://localhost:5000
+cypress run --record --key a0cba5c6-0650-4abe-8d41-d990bb7a0a66
```

Comment:
On the fence about this one. If we use an ENV var, then recordings won't work for pull requests from forks because Travis doesn't inject env into builds from forks. Also this string is more of a project identifier than an actual secret someone can use to do bad things. I guess they a bad actor could inject recordings into our account but that seems unlikely.. what would be the point?

How about we roll with this for now: If we see any abuse we can roll the creds and envify.